aio/content/marketing/contributors.json

```
@@ -909,5 +909,13 @@
     "website": "https://medium.com/@abampakos",
     "bio": "Aristeidis is a front end web developer at Plex-Earth working with Angular. He is a co-organizer of the Angular Athens meetup and the author of Learning Angular - 3rd Edition. He is also a senior Angular instructor at Code.Hub where he teaches the Angular framework to other developers.",
     "groups": ["GDE"]
+  },
+  "phenomnominal": {
```

Please change the key to be your GitHub handle :pray:

```suggestion
  "phenomnomnominal": {
```

---

Oops good spot @gkalpak

---

Oops my bad! Thanks!